models/asana__task.sql

```diff
@@ -5,7 +5,7 @@ with task as (
 
 task_comments as (
 
-    select * 
```

Comment:
Apologies for the extra diffs, it was auto whitespace trimming in my editor.  I can put these back if you want!